### PR TITLE
crypto modules: fix sanity errors

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fixes sanity checks for `crypto`/`acme` modules/files.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lib/ansible/module_utils/acme.py
lib/ansible/module_utils/crypto.py
lib/ansible/modules/crypto/certificate_complete_chain.py
lib/ansible/modules/crypto/luks_device.py
lib/ansible/modules/crypto/openssh_cert.py
lib/ansible/modules/crypto/openssl_csr.py
lib/ansible/modules/crypto/openssl_privatekey.py
lib/ansible/modules/crypto/openssl_privatekey_info.py
lib/ansible/modules/crypto/acme/acme_certificate.py
lib/ansible/modules/crypto/acme/acme_challenge_cert_helper.py
lib/ansible/modules/crypto/certificate_complete_chain.py
lib/ansible/modules/crypto/openssh_cert.py
lib/ansible/modules/crypto/openssl_csr_info.py
lib/ansible/modules/crypto/openssl_privatekey.py
lib/ansible/modules/crypto/openssl_publickey.py

### CODE REVIEW

File: lib/ansible/module_utils/acme.py [hunk 1]
```diff
@@ -230,27 +230,27 @@ def _parse_key_openssl(openssl_binary, module, key_file=None, key_content=None):
...
             alg = 'ES512'
-            hash = 'sha512'
+            hashalg = 'sha512'
             point_size = 66
             curve = 'P-521'
         else:
             return 'unknown elliptic curve: %s / %s' % (asn1_oid_curve, nist_curve), {}
-        bytes = (bits + 7) // 8
-        if len(pub_hex) != 2 * bytes:
+        no_bytes = (bits + 7) // 8
```
Comment (MarkusTeufelberger) [inline lib/ansible/module_utils/acme.py]: How about `num_bytes`? `no` sounds a bit "negative" to me.

Comment (felixfontein) [reply lib/ansible/module_utils/acme.py]: Sure, fine for me.

**Comment by felixfontein** [comment]
Reopening to run tests again.

ready_for_review

**Comment by MarkusTeufelberger** [comment]
Hm, needs a rebase apparently, then I think it's fine to merge.

**Comment by felixfontein** [comment]
Rebased to remove conflicts.

**Comment by felixfontein** [comment]
Merging since CI ran through fine :)

**Comment by felixfontein** [comment]
@MarkusTeufelberger thanks for reviewing this!

I'll now rebase the other PRs so no new sanity failures get merged in accidentally.

**Pull request merged**